tensordiffeq/models.py

```diff
@@ -13,18 +13,23 @@ class CollocationSolverND:
     def __init__(self, assimilate=False, verbose=True):
         self.assimilate = assimilate
         self.verbose = verbose
+        self.losses = []
 
-    def compile(self, layer_sizes, f_model, domain, bcs, isAdaptive=False,
-                dict_adaptive=None, init_weigths=None, g=None, dist=False):
+    def compile(self, layer_sizes, f_model, domain, bcs, Adaptive_type=None,
+                dict_adaptive=None, init_weights=None, g=None, dist=False):
         """
         Args:
             layer_sizes: A list of layer sizes, can be overwritten via resetting u_model to a keras model
             f_model: PDE definition
             domain: a Domain object containing the information on the domain of the system
             bcs: a list of ICs/BCs for the problem
-            isAdaptive: Boolean value determining whether to implement self-adaptive solving
+            Adaptive_type: string with the adaptive method
+                                1 - Self-adaptive (https://arxiv.org/pdf/2009.04544.pdf),
+                                2 - Self-adaptive_loss with weights for the entire loss function,
+                                3 - NTK
+                                4 - None (no adaptive method)
```

Comment:
Should we make this into a dictionary so users select an int value such as '4' and it selects the scheme? that way we don't have to worry about spelling, etc.